packages/editor/src/components/rich-text/formatting-controls.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { displayShortcut } from '@wordpress/keycodes';
+
+export const DEFAULT_CONTROLS = [ 'bold', 'italic', 'strikethrough', 'link', 'code' ];
```

Comment:
@iseulde I moved those contants to its own file. It's now easy to notice that code is listed as a default control, but it's not provided in formatting controls. Any thoughts?

Reply:
I'm also not sure why there are two constants, and why `code` is there... `code` does have a shortcut and a pattern, but this constant is not used there.

Reply:
yes, let's remove it.